contract.py

```diff
@@ -109,9 +112,13 @@ def fill(self, total: arc4.UInt64, funding: arc4.UInt64) -> None:
     ##############################################
     @arc4.abimethod
     def participate(self, vote_k: Bytes, sel_k: Bytes, vote_fst: arc4.UInt64, vote_lst: arc4.UInt64, vote_kd: arc4.UInt64, sp_key: Bytes) -> None: 
-        self.enforce_step(UInt64(3)) # Full
-        self.require_owner()
-        self.require_payment(self.owner, UInt64(1000))
+        ###########################################
+        assert self.funding > 0, "funding initialized"
+        ###########################################
+        assert Txn.sender == self.owner, "must be owner" 
+        ###########################################
+        assert self.require_payment(self.owner) == UInt64(1000), "payment amout accurate"
```

Comment:
I think there is a `global` value that is the fee amount, but I'm not positive and am too lazy to check.